Components/ArticleDetails.jsx

```diff
@@ -14,15 +18,17 @@ export default function ArticleDetails () {
         })
     }, [article_id])
 
+    useEffect(() => {
+        getCommentsById(article_id).then(( comments ) => {
+            setCurrentComments(comments)
+        })
+    }, [article_id])
+
```

Comment:
Would be better having this useEffect in the ArticleComments component. That way, the component can fetch its own comments rather than fetching them here and then passing them down